README.md

```diff
@@ -12,3 +12,102 @@ Due to being deployed in different partner's environments, and us not
 always having full administrative control of that environment, each
 backend is different in some way. We do try to minimise these
 differences, but they are unavoidable.
+
+## usage
+
+To manage a backend run the script BACKEND/manage.sh as root. e.g.:
+
+    sudo ./tpp/manage.sh
+
+This will ensure the right packages, users, groups and configuration is set up
+on that backend. 
+
+Directory layout:
+
+./scripts 	    - various bash scripts to perform specific actions, designed to
+            	  be idempotent
+./tpp     	    - scripts for tpp backend
+./emis    	    - scripts for emis backend
+./keys/$USER 	- public keys to add to ssh for $USER
+developers      - developers gh account list. To be replaced by
+	              authenticated API call to job-server later, perhaps.
+
+./tests         - test scripts
```

Comment:
this formatting looks a bit weird - my personal preference for this sort of thing is markdown lists with indentation, e.g.:

```
* ./scripts
  * various bash scripts
* ./tpp 
  * scripts for tpp
...etc...
```

Reply:
Fixed